xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.cpp

```diff
@@ -1248,6 +1254,7 @@ void CLinuxRendererGL::RenderFromFBO()
 {
   glEnable(GL_TEXTURE_2D);
   glActiveTextureARB(GL_TEXTURE0);
+  glBindTexture(GL_TEXTURE_2D, m_fbo.fbo.Texture());
```

Comment:
that looks wrong. we render from fbo that is loaded into texture units. what is your intent?

Reply:
It is present in the GLES renderer however it doesn't seem to effect anything with or without it so I removed it now.